Custom-Audio-Driver/Custom-Audio-Driver/DefaultAudioDevice.swift

```diff
@@ -630,20 +632,33 @@ func updatePlayoutDelay(withAudioDevice audioDevice: DefaultAudioDevice) {
     audioDevice.playoutDelayMeasurementCounter += 1
     if audioDevice.playoutDelayMeasurementCounter >= 100 {
         // Update HW and OS delay every second, unlikely to change
-        audioDevice.playoutDelay = 0
+        var tempPlayoutDelay: UInt32 = 0
         let session = AVAudioSession.sharedInstance()
         
         // HW output latency
         let interval = session.outputLatency
-        audioDevice.playoutDelay += UInt32(interval * 1000000)
+        tempPlayoutDelay += UInt32(interval * 1000000)
         // HW buffer duration
         let ioInterval = session.ioBufferDuration
-        audioDevice.playoutDelay += UInt32(ioInterval * 1000000)
-        audioDevice.playoutDelay += UInt32(audioDevice.playoutAudioUnitPropertyLatency * 1000000)
+        tempPlayoutDelay += UInt32(ioInterval * 1000000)
+        tempPlayoutDelay += UInt32(audioDevice.playoutAudioUnitPropertyLatency * 1000000)
         // To ms
-        audioDevice.playoutDelay = (audioDevice.playoutDelay - 500) / 1000
+        if( tempPlayoutDelay >= 500 ) {
```

Comment:
Why do we use a StandardPlayoutDelay now?

Reply:
In the scenario that tempPlayoutDelay is lower than 500, to avoid having a negative value which will crash the system, we assign the default playoutDelay value (15). 

This value comes from my tests. I have tested several scenarios, including default audio driver embedded in multi-party session and the value audioDevice.playoutDelay is assign is always 15

What would you suggest? Skip and let it calculate in the next iteration?

Reply:
Ok, seems that we can remove kStandardPlayoutDelay now.